#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_disable_epson2.dpatch by  <jblache@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Disable the epson2 backend by default.

@DPATCH@
diff -urNad sane-backends-1.0.19~/backend/dll.conf.in sane-backends-1.0.19/backend/dll.conf.in
--- sane-backends-1.0.19~/backend/dll.conf.in	2008-01-18 02:27:22.000000000 +0100
+++ sane-backends-1.0.19/backend/dll.conf.in	2008-02-12 21:18:36.597390953 +0100
@@ -21,7 +21,7 @@
 dmc
 epjitsu
 epson
-epson2
+#epson2
 fujitsu
 #gphoto2
 genesys
